charms/istio-pilot/tests/unit/test_charm.py

```diff
@@ -29,17 +33,17 @@ def side_effect(*args, **kwargs):
 
 
 def test_events(harness, mocker):
-    mocker.patch('lightkube.codecs.load_all_yaml')
-    mocker.patch('resources_handler.load_in_cluster_generic_resources')
+    mocker.patch("lightkube.codecs.load_all_yaml")
+    mocker.patch("resources_handler.load_in_cluster_generic_resources")
 
     harness.set_leader(True)
     harness.begin_with_initial_hooks()
 
-    install = mocker.patch('charm.Operator.install')
-    remove = mocker.patch('charm.Operator.remove')
-    send_info = mocker.patch('charm.Operator.send_info')
-    handle_ingress = mocker.patch('charm.Operator.handle_ingress')
-    handle_ingress_auth = mocker.patch('charm.Operator.handle_ingress_auth')
+    install = mocker.patch("charm.Operator.install")
```

Comment:
did you run black on the charm code or is this introduced by the backported charm code?

Reply:
Yeah you mean because of the changes in `'` to `"`?  somewhere between when this branch departed from main and now, `black` has been run on main, so these quote changes are coming from main via the backport.  They aren't necessary, but I took them in mainly so that I didn't introduce an error in the backporting.  They can be removed

Reply:
Yeh, I just wanted to know as I was suspecting some other changes were introduced. We can keep them as long as it is just the formatting.